free-3.md

```diff
@@ -51,18 +60,19 @@ This section examines whether the idea makes for a good CS 111 project.
 
 
 ### Suitability
-_If someone were to work on this project, what percentage of their time would be
-spent directly engaging in the **language** aspects of this project (e.g.,
-making language design decisions), as opposed to "systems" aspects of the
-project (e.g., implementing a complicated semantics that doesn't require a lot
-of language design)?_
+I think the work would be split up pretty evenly between implementation 
+and language design.
 
 
 ### Scope
-_How big an idea is this? How ambitious is this project?_
+I don't think this is a particularly ambitious project. It would just require
+knowledge of how to block apps on a system which is clearly possible given
+the above mentioned apps that already do this.
```

Comment:
It could still be a bit ambitious if you wanted the language itself to have more features that make use of the semantics in different ways (you might want to handle repeating vs. one-time schedule events, for example). I don't mean this as a criticism but rather I mean to suggest that you could do a decent amount with it.